### DISCUSSION TITLE
Licence Query

### DISCUSSION THREAD

**Discussion by InfoSPM** [community/community#53936] [Repositories]
If members of our organization no longer require access to any repositories owned by the organization, if we remove them from the organization does the licenses will be remained or will get deleted?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @InfoSPM, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by byrneh** [comment]
@InfoSPM and @Jakkpob 

If they are no longer an Organization owner/members (including dormant users) and they are
not an outside collaborators on private repositories owned by your organization, excluding forks
or anyone anyone with a pending invitation to become an outside collaborator on private or internal repositories owned by your organization, excluding forks
A license will not be consumed.

A reference link provided of further information should you need it [#people-that-consume-a-license](https://docs.github.com/en/billing/managing-billing-for-your-github-account/about-per-user-pricing#people-that-consume-a-license)
